testdata/translation/config.js

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "translation-test",
+  "paths": ["."],
+  "inputs": "main.js",
+
+  "mode": "ADVANCED",
+
+  "translations": ".",
+  "language": "en"
```

Comment:
That was my original idea, yes. Each lang gets a separate output (or set of
outputs, with modules), and is a separate compilation, so it makes sense to
have a separate config.
On Mar 16, 2016 8:53 AM, "Nick Santos" <notifications@github.com> wrote:

> In testdata/translation/config.js
> <https://github.com/bolinfest/plovr/pull/95#discussion_r56359616>:
>
> > @@ -0,0 +1,10 @@
> > +{
> > +  "id": "translation-test",
> > +  "paths": ["."],
> > +  "inputs": "main.js",
> > +
> > +  "mode": "ADVANCED",
> > +
> > +  "translations": ".",
> > +  "language": "en"
>
> is the idea of this pr that you would have a separate config for each
> language?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/bolinfest/plovr/pull/95/files/dc127ed0b26949d173fe76477d677c87d64f2ee4#r56359616>
>